core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -259,9 +260,25 @@ public UfsStatus[] listStatus(String path, ListOptions options) throws IOExcepti
     return statuses;
   }
 
+  /**
+   * @param fi the FileInfo of this file. Cached pages are identified by PageId
+   * @return true at this moment
+   */
+  @Override
+  public boolean invalidateCachedFile(FileInfo fi) {
```

Comment:
I might prefer to use the full name rather than abbr in java.  
```suggestion
  public boolean invalidateCachedFile(FileInfo fileInfo) {
```

Reply:
OK.

Reply:
Fixed.